library/components/TTooltip.vue

```diff
@@ -33,20 +37,22 @@ export default {
   props: {
     width: {
       type: String,
-      default: "200px"
+      default: "200px",
     },
     position: {
       type: String,
-      default: "top"
+      default: "top",
     },
-    isOpen: { // To control state from a component.
+    isOpen: {
+      // To control state from a component.
       type: Boolean,
       default: false,
-    }
+    },
   },
   data: () => ({
     positions: { arrow: {}, tooltip: {} },
     isVisible: this.isOpen,
+    isTooltipFocused: false, // If cursor is on tooltip.
```

Comment:
Change 2:
This will allow us to keep our tooltip opened if user moves cursor on it.

Reply:
Non-blocking: I preferred it when we had one data property that reflects the current visibility of the component. It felt like that was a single source of truth. Now it feels like there are two, which makes it harder to reason about the component.